src/components/test/Countries.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../../redux/configStore';
+import getCountries from '../../redux/api';
+
+import ContriesHomepage from '../ContriesHomepage';
+
```

Comment:
- Please add unit tests for several components of the app and some pure functions in Redux code.